snmp/datadog_checks/snmp/snmp.py

```diff
@@ -500,15 +500,20 @@ def get_index_tags(
 
         for column_tag in column_tags:
             raw_column_value = column_tag.column
-
+            self.log.trace(
+                'Processing column tag: raw_column_value=%s index_slices=%s', raw_column_value, column_tag.index_slices
+            )
             if column_tag.index_slices:
                 new_index = transform_index(index, column_tag.index_slices)
             else:
                 new_index = index
+            self.log.trace('Processing column tag: new_index=%s old_index=%s', new_index, index)
+            if new_index is None:
+                continue
             try:
                 column_value = results[raw_column_value][new_index]
             except KeyError:
-                self.log.warning(
+                self.log.debug(
```

Comment:
Note: we need to use debug since this might happen when there are 2 rules for the same tag, like `pdu_name` above.